pkg/database/vercode.go

```diff
@@ -207,6 +207,13 @@ func (db *Database) SaveVerificationCode(vc *VerificationCode, maxAge time.Durat
 	return db.db.Save(vc).Error
 }
 
+// ExpireCode saves a verification code as expired.
+func (db *Database) ExpireCode(vc *VerificationCode) error {
```

Comment:
this should be in a transaction and to a for update select and make sure it makes sense to actually invalidate the code

Reply:
excellent point. I think I got excited to see this work at all...